### PR TITLE
feat: Bump kube-prometheus-stack to 48.3.2

### PR DESCRIPTION
**Pull request by Azhovan**
**What problem does this PR solve?**:
Back port to 2.6
chart changes: https://github.com/mesosphere/charts/pull/1448

**Which issue(s) does this PR fix?**:
https://d2iq.atlassian.net/browse/D2IQ-98716

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: services/kube-prometheus-stack/48.3.2/helmrelease/kube-prometheus-stack.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 46.8.0
+      version: 48.3.2
```
Comment (gracedo) [inline services/kube-prometheus-stack/48.3.2/helmrelease/kube-prometheus-stack.yaml:14]: I am just worried about jumping 2 major versions in a dkp patch version. in the past we have created "release branches" for the helm charts in `m/charts` but I would have to remember how that process works, it was a little manual and a little risky when publishing the version

Comment (gracedo) [reply services/kube-prometheus-stack/48.3.2/helmrelease/kube-prometheus-stack.yaml:14]: I tried this here https://github.com/mesosphere/charts/pull/1449 backporting the fix to the KPS version we released in DKP 2.6.0 -- some more details in the PR description

File: services/kube-prometheus-stack/48.3.2/etcd-metrics-proxy.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ spec:
   wait: true
   interval: 6h
   retryInterval: 1m
-  path: ./services/kube-prometheus-stack/46.8.0/etcd-metrics-proxy
+  path: ./services/kube-prometheus-stack/48.3.2/etcd-metrics-proxy
```
Comment (gracedo) [inline services/kube-prometheus-stack/48.3.2/etcd-metrics-proxy.yaml:12]: Let's use 46.8.1 now that it has been published

**Comment by Azhovan** [comment]
closed in favor of https://github.com/mesosphere/kommander-applications/pull/1562

**Pull request closed without merging**